build_projects.py

```diff
@@ -0,0 +1,100 @@
+#!/bin/python
+
+import argparse
+import json
+import os
+import multiprocessing
+import sys
+
+TGREEN =  '\033[32m' # Green Text	
+TBLUE =  '\033[34m' # Green Text	
+TRED =  '\033[31m' # Red Text	
+TWHITE = '\033[39m' #Withe text
+
+description_help='''Build all adicup projects
+Examples:\n
+    >python build_projects.py ..\.. aducm3029
+	>python build_projects.py /home/user/noos xilinx
+'''
+
+def parse_input():
+    parser = argparse.ArgumentParser(description=description_help,\
+			    formatter_class=argparse.RawTextHelpFormatter)
+    parser.add_argument('adicup_location', help="Path to adicup repo location")
+    parser.add_argument('export_dir', help="Path where to save files")
+    args = parser.parse_args()
+
+    return (args.adicup_location, args.export_dir)
+
+VERBOSE = 0
+ERR = 0
+
+def run_cmd(cmd):
+	log_file = 'log.txt'
+	print(cmd)
+	sys.stdout.flush()
+	if (VERBOSE):
+		os.system(cmd)
+	else:
+		err = os.system(cmd + ' > %s 2>&1' % log_file)
+		if (err != 0):
+			global ERR
+			print(TRED + "Error" + TWHITE)
+			print("See log:")
+			os.system("cat %s" % log_file)
+			ERR = err
+
+def to_blue(str):
+	return TBLUE + str + TWHITE
+
+def print_build(build_name, project):
+	print("Runing build %s on project %s" % (to_blue(build_name),
+						to_blue(project)))
+
+def build_cces_project(adicup_location, project, project_dir, export_dir):
+	DEFAULT_WORKSPACE = os.path.join(adicup_location, 'workspace')
+	CCES_TEMPLATE = "cces -nosplash \
+-application com.analog.crosscore.headlesstools -data %s -project %s "
+	print_build("cces", project)
+
+	cces_cmd = CCES_TEMPLATE % (DEFAULT_WORKSPACE, project_dir)
+	run_cmd(cces_cmd + '-build Debug')
+
+	binary = os.path.join(project_dir, 'Debug', project)
```

Comment:
Release

Reply:
No all projecte worked on release, that's why I choosed Debug

Reply:
The projects should all work in release and debug as long as optimization flags are the same (on the O0 level).